### PR TITLE
transitions the game into the correct era

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
transitions the game into the correct era (500 to 1500 AD), including job name changes, and the removal of all guns
several other non-important changes can be found in the changelog

image

## Why It's Good For The Game
fixes basically everything wrong with the game

## Changelog
:cl:
balance: ian was found to be too weak as an endgame boss and now has a mounted pulse carbine turret on his back
add: tableslamming fat people breaks the table
refactor: stamina has been removed, existing stamina effects now work off your nutrition
del: deletes boxstation in favour of rectanglestation
add: seven undocumented changes
fix: fixes something while breaking two other things
add: moths now get flung uncontrollably into nearby bright light sources
add: the entire map is now inside a vore belly object to optimise the vore subsystem
del: removed kevin
imageadd: added several libraries of copyrighted images
add: lizards and felinids are now unable to vote on the gamemode and next map
del: deletes modular citadel
add: to enhance roleplay, you can now opt out of death
del: cyborgs have been removed
add: medibots have been added as a playable species
add: felinids now uncontrollably vomit hairballs at random times
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/chief_medical_officer.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/job/cmo
-	title = "Chief Medical Officer"
+	title = "Head Surgeon"
```
Comment (The-Sun-In-Splendour) [inline code/modules/jobs/job_types/chief_medical_officer.dm]: Change this to Head Chirurgeon.

File: code/modules/jobs/job_types/medical_doctor.dm [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /datum/job/doctor
-	title = "Medical Doctor"
+	title = "Surgeon"
```
Comment (The-Sun-In-Splendour) [inline code/modules/jobs/job_types/medical_doctor.dm]: Chirurgeon.

File: code/modules/jobs/job_types/medical_doctor.dm [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 	flag = DOCTOR
 	department_head = list("Chief Medical Officer")
 	department_flag = MEDSCI
 	faction = "Station"
 	total_positions = 5
 	spawn_positions = 3
-	supervisors = "the chief medical officer"
+	supervisors = "the head surgeon"
```
Comment (The-Sun-In-Splendour) [inline code/modules/jobs/job_types/medical_doctor.dm]: head chirurgeon.

**Comment by The-Sun-In-Splendour** [comment]
I suggest changing the name of AI to "God."

File: code/modules/jobs/job_types/chief_medical_officer.dm (hunk 1 above)
Comment (Solaris-Shade) [reply code/modules/jobs/job_types/chief_medical_officer.dm]: Ooooo thats a good one.

File: code/modules/jobs/job_types/geneticist.dm [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-	title = "Geneticist"
+	title = "Witch Doctor"
 	flag = GENETICIST
 	department_head = list("Chief Medical Officer", "Research Director")
 	department_flag = MEDSCI
 	faction = "Station"
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "the chief medical officer and research director"
+	supervisors = "the head surgeon and head scholar"
```
Comment (Solaris-Shade) [inline code/modules/jobs/job_types/geneticist.dm]: head chirurgeon. The-Sun-In_Splendor's suggestion.

File: code/modules/jobs/job_types/paramedic.dm [hunk 5]
```diff
@@ -1,12 +1,12 @@
 /datum/job/paramedic
-	title = "Paramedic"
+	title = "Field Surgeon"
```
Comment (Solaris-Shade) [inline code/modules/jobs/job_types/paramedic.dm]: Paramedic name suggestion - Leecher.

File: code/modules/jobs/job_types/warden.dm [hunk 6]
```diff
@@ -1,13 +1,13 @@
 /datum/job/warden
-	title = "Warden"
+	title = "Jailer"
```
Comment (The-Sun-In-Splendour) [inline code/modules/jobs/job_types/warden.dm:2]: Suggest changing this to "Gaoler"
Which is the archaic way of saying Jailer.

Comment (timothyteakettle) [reply code/modules/jobs/job_types/warden.dm:2]: harder to understand what the role actually is from 'gaoler' vs 'jailer' to 'warden' so i'm not gonna do that, neat suggestion though

File: code/modules/jobs/job_types/paramedic.dm (hunk 5 above)
Comment (timothyteakettle) [reply code/modules/jobs/job_types/paramedic.dm]: see my response to the other review about changing jailer to gaoler

File: code/modules/jobs/job_types/assistant.dm [hunk 7]
```diff
@@ -31,6 +31,9 @@ Assistant
 /datum/outfit/job/assistant
 	name = "Assistant"
 	jobtype = /datum/job/assistant
+	head = /obj/item/clothing/head/helmet/knight
+	suit = /obj/item/clothing/suit/armor/riot/knight
+	backpack_contents = list(/obj/item/claymore/april_fools=1,)
```
Comment (PersianXerxes) [inline code/modules/jobs/job_types/assistant.dm:36]: How did a lowly peasant afford any of this? smh, my immersion

Comment (timothyteakettle) [reply code/modules/jobs/job_types/assistant.dm:36]: they hit a guard with a toolbox and looted it from them

File: code/modules/jobs/job_types/security_officer.dm [hunk 8]
```diff
@@ -1,13 +1,13 @@
 /datum/job/officer
-	title = "Security Officer"
+	title = "Guard"
```
Comment (PersianXerxes) [inline code/modules/jobs/job_types/security_officer.dm]: Watchman* & Watch Captain*
smh, you talking goo.

**Comment by PersianXerxes** [comment]
I'd say assistants should be squires, but then they'd think they actually have rights. :pensive:

**Comment by timothyteakettle** [comment]
i was thinking squire but i thought calling them peasants was too funny to pass up

**Comment by PersianXerxes** [comment]
Too much effort for this, but word filter that replaces all instances of "assistant" in say text with "squire, knave, or peasant" :^)

**Comment by Hartmann-von-Berlichingen** [comment]
woa

**Comment by TheEaterOfBeans** [comment]
> woa

HARTMANN DETECTED

**Pull request closed without merging**